grasa_event_locator/views.py

```diff
@@ -343,6 +347,7 @@ def denyEvent(request, eventID):
         if request.user.is_authenticated and request.user.userinfo.isAdmin and request.user.userinfo.isActive:
                 p = Program.objects.get(pk=eventID)
                 p.delete()
+                print(send_email([str(User.objects.get(pk=p.user_id.user_id))], "GRASA - Event Denied", "Your edited event has been denied. Contact GRASA for details."))
```

Comment:
This says your edited event,  should just be your event

Reply:
Altered in a subsequent commit.